modules/40_techpol/NDC/equations.gms

```diff
@@ -8,15 +8,17 @@
 
 *' @equations
 
-q40_ElecBioBound(t,regi)$(t.val gt 2015)..
+
 ***am minimum targets for certain technologies
+q40_ElecBioBound(t,regi)$(t.val gt 2015)..
     sum(te2rlf(te,rlf)$(sameas(te,"biochp") OR sameas(te,"bioigcc") OR sameas(te,"bioigccc")), vm_cap(t,regi,te,rlf))
-      * 1000 =g= p40_ElecBioBound(t,regi);	 
+      =g= p40_ElecBioBound(t,regi) * 0.001
+;	 
 
+*** windoffshore-todo: separate onshore and offshore NDC targets
 q40_windBound(t,regi)$(t.val gt 2025)..
-***making target apply to both wind onshore and offshore, when offshore switch is on
-sum(teWind(te), vm_cap(t,regi,te,"1")) 
-    =g= p40_TechBound(t,regi,"wind")*0.001
+  sum(teWind(te)$(sameas(te,"windon") OR sameas(te,"windoff")), vm_cap(t,regi,te,"1")) 
+    =g= p40_TechBound(t,regi,"windon") * 0.001
```

Comment:
Actually, I find it potentially confusing if you read in a "all-wind target" as "windon" and then apply it to all wind. 

so if you keep "wind" as an empty technology, this makes it very easy for you - just read in the all-wind target for the technology "wind", and then apply it to the sum of both. 

so 

 sum(teWind(te)$(sameas(te,"windon") OR sameas(te,"windoff")), vm_cap(t,regi,te,"1")) 
    =g= p40_TechBound(t,regi,"wind") * 0.001

Reply:
Confirmed that bounds are for the sum of windOn and windOff:
https://github.com/pik-piam/mrremind/blob/10d5f851e92778814892af4216cc222c6431718c/R/convertREN21.R#L35

> [mrremind](https://github.com/pik-piam/mrremind/tree/10d5f851e92778814892af4216cc222c6431718c)/[R](https://github.com/pik-piam/mrremind/tree/10d5f851e92778814892af4216cc222c6431718c/R)/convertREN21.R
>   ```r
> x_tmp[, , "Wind"] <- x_tmp[, , "Wind_ON"] + x_tmp[, , "Wind_OFF"]
>   ```

Let's use `pewin` for such bounds, which then offers a choice: either we update mrremind to also output pewin bounds, or we re-separate the windon and windoff bounds (hopefully without to much trouble).

Reply:
Edit: cannot use `pewin` as it is not in all_te. I keep wind and make the bounds ready for mrremind changes:
- as long as there is a `wind` bound, it applies on windon+windoff
- if there is a `windon` bound, then it applies on windon
- if there is a `windoff` bound, then it applies on windoff